tests/sqllogictests/suites/ee/01_ee_system/01_0003_stream.test

```diff
@@ -250,9 +250,43 @@ query T
 select * from s_14062 where change$row_id = '1';
 ----
 
+statement ok
+drop stream s_14062
+
+statement ok
+drop table t_14062 all
+
 ######################
 # end of issue 14062 #
 ######################
 
+###############
+# issue 14099 #
+###############
+
+statement ok
+create table t_14099 (a int, b int);
+
+statement ok
+create stream s_14099 on table t_14099 ;
+
+statement ok
+insert into t_14099 values(1, 1), (2, 2);
+
+query T
+select * from s_14099 where a>1 limit 1;
```

Comment:
What will happend if we do:
```
insert into t_10499 select * from s_14099 limit 1;
```
What's the status of s_14099?

Reply:
> ```
> insert into t_10499 select * from s_14099 limit 1;
> ```
s_14099 contain `(1, 1), (2, 2)` values